browser/src/core/Debug.js

```diff
@@ -515,13 +517,28 @@ L.DebugManager = L.Class.extend({
 		}
 
 		// Toggle sidebar
+		if (!this._map._docLoadedOnce) {
+			// When first opening the document, initializeSidebar is called
```

Comment:
good catch =) I hate 200ms delays though - possibly we should rework the sidebar delay logic; and/or put this in initializeSidebar itself ? =) I'm not optimistic about racing timers on loaded machines.